converter/pom.xml

```diff
@@ -111,5 +111,36 @@
 			<artifactId>cloning</artifactId>
 			<version>1.9.10</version>
 		</dependency>
+
+        <dependency>
```

Comment:
these deps should all be automatically imported from test-commons, right?

Reply:
They do not get pulled in properly with Eclipse JUnit engine. Not Sure why. But yes, this is not a problem with command line maven.